config.js

```diff
@@ -41,6 +41,42 @@ config.mqtt = {
      * consists of a series of parameters separated by commas.
```

Comment:
CHANGES_NEXT_RELEASE entry about the support to HTTPS should be added.

Reply:
Fixed 6fd6ea9